src/genie/libs/parser/asa/tests/ShowInventory/cli/equal/golden_output_output.txt

```diff
@@ -1,5 +1,3 @@
-
-ciscoasa> show inventory
```

Comment:
why these lines been removed from the Unittest?

Reply:
this is ok. we can have only show command output excluding command itself.

Reply:
I requested that, it is a command line prompt and not device output, it's better to not include that.